components/stacks-network/src/orchestrator.rs

```diff
@@ -346,30 +347,46 @@ impl DevnetOrchestrator {
             "initializing",
         );
 
-        send_status_update(
-            &event_tx,
-            enable_subnet_node,
-            &self.logger,
-            "stacks-api",
-            Status::Red,
-            "initializing",
-        );
-        send_status_update(
-            &event_tx,
-            enable_subnet_node,
-            &self.logger,
-            "stacks-explorer",
-            Status::Red,
-            "initializing",
-        );
-        send_status_update(
-            &event_tx,
-            enable_subnet_node,
-            &self.logger,
-            "bitcoin-explorer",
-            Status::Red,
-            "initializing",
-        );
+        if !disable_stacks_api {
+            send_status_update(
+                &event_tx,
+                enable_subnet_node,
+                &self.logger,
+                "stacks-api",
+                Status::Red,
+                "initializing",
+            );
+        }
+        if !disable_postgres {
+            send_status_update(
+                &event_tx,
+                enable_subnet_node,
+                &self.logger,
+                "postgres",
+                Status::Red,
+                "initializing",
+            );
+        }
```

Comment:
The postgres status updates can be removed if we don't show it in the UI right?

Reply:
Ah, I see.
Then the `send_status_update` function would need to be revamped because if no service name is matched, it does nothing I believe